src/basic/missing_capability.h

```diff
@@ -5,22 +5,30 @@
 
 /* 3a101b8de0d39403b2c7e5c23fd0b005668acf48 (3.16) */
 #ifndef CAP_AUDIT_READ
-#  define CAP_AUDIT_READ 37
+#    define CAP_AUDIT_READ 37
```

Comment:
Hmm, why are these indented even further?

Reply:
Oops, autoprocessing failure. Should be fixed now.